src/lambda/commands/deploySamApplication.ts

```diff
@@ -112,7 +112,7 @@ export async function deploySamApplication(
 
         await deployApplicationPromise
         // no need to await, doesn't need to block further execution
-        vscode.commands.executeCommand('aws.refreshAwsExplorer')
+        vscode.commands.executeCommand('aws.refreshAwsExplorerNode')
```

Comment:
Calling with no argument is the same as refreshing the root name. Maybe a constant set to `undefined` would read better?

Reply:
I feel like this has a bit of a loss of clarity (this has no node context and thus has to refresh the whole explorer). Would it make more sense to have `aws.refreshAwsExplorer` have an `emitTelemetry: boolean = true` param and have this call with an explicit false?